pkg/cli/cli.go

```diff
@@ -596,6 +603,8 @@ func connectionCommands(opts *subCommandOpts) []*cobra.Command {
 	connect.Flags().BoolVarP(&connFlags.skipConfirmation, "yes", "y", false,
 		"Don't ask for confirmation for closing multiple connections.")
 	connect.Flags().StringVar(&connFlags.ice_config, iceConfigFlag, "", iceConfigFlagDesc)
+	connect.Flags().BoolVar(&connFlags.doNotUseWebrtc, "do_not_use_webrtc", false,
```

Comment:
`do_not_use_webrtc` is a mouthful, maybe this could be `--webrtc` with default to true, and a complementary one like `--ssh_port_forwarding`.

Alternatively, this could be a single flag like `--transport=webrtc|ssh|...`

Reply:
+1 for @jemoreira 's comment. it would be --transport=port_forwarding or something(because it doesn't use ssh directly)

Reply:
cvdr does port forwarding between docker ip:<adb port> and local unix domain socket file(like `socat UNIX-LISTEN:/tmp/my.sock,fork TCP:<docker internal ip>:6520`) via socks5 proxy

Reply:
I see, then `port_forwarding` or even leaving it empty are better than my suggestion.

Reply:
I understood your concern. At least, I won't set naming of flag as `do_not_use_webrtc`.

However, I think neither `ssh` nor `port_forwarding` is the good naming of this because:
- We don't use `ssh` when not using SOCKS5 proxy
- I don't think we can say this as port forwarding, since an endpoint is not port, just a file-based socket.

So I just want to set the name as `proxy`. However if you have any concern on the naming, please let me know :) I think changing it again would be very simple task.

Reply:
Done for this. I'll resolve this conversation tomorrow.